vrecord

```diff
@@ -2007,6 +2014,7 @@ _review_all_options(){
     fi
     _review_option -n -d "digital" "WAVEFORM_SCALE_CHOICE" "${WAVEFORM_SCALE_OPTIONS[@]}"
     _review_option -n -d "Broadcast Range" "SIGNAL_VIEW_CHOICE" "${SIGNAL_OPTIONS[@]}"
+    _review_option -n -d "As is" "SIGNAL_INT_CHOICE" "${INT_OPTIONS[@]}"
```

Comment:
This line breaks things for me as `_review_option` will error by saying "As is" is not a valid option and exit. Does this work on your end?

Reply:
changing `"As is"` to "auto" fixes this on my end - this settings for this option do not, however, persist currently and are not apparently stored in the config file - was this intentional?

Reply:
I switched to 'auto'